cmake_format/commands.py

```diff
@@ -511,11 +525,26 @@ def get_fn_spec():
           "RUN_OUTPUT_VARIABLE": ZERO_OR_MORE
       })
 
+  fn_spec.add(
+      "write_basic_package_version_file", flags=[], kwargs={
+          "COMPATIBILITY": 1,
+          "VERSION": 1
+      })
+
   fn_spec.add(
       "if", flags=[], kwargs={
           "NOT": ONE_OR_MORE,
           "AND": ONE_OR_MORE,
-          "OR": ONE_OR_MORE
+          "OR": ONE_OR_MORE,
```

Comment:
I know its odd but looking at a few instances of complex boolean statements in my own listfiles, the formatting generally looks better if most of these act as "flags" even though they are really keywords. This is what I do in `parser.py` currently for conditional groups. It makes me think the formatting rules should be made more robust or configurable, but for now I'm going to do the same for the if spec.